gps/turn/index.html

```diff
@@ -97,9 +97,9 @@ <h2 class="text-left">Support Contacts</h2>
   <span class="one-of-two">
     <h2 class="text-left">Related Resources</h2>
     <ul>
-      <li><a href="{% link gps/images/TURN_Network_8x11_2019.jpg %}" rel="noopener" target="_blank">TURN GPS Network Map
+      <li><a href="{% link images/2023TURNGPSNetwork85x11.png %}" rel="noopener" target="_blank">TURN GPS Network Map
           8.5 x 11</a>
-      <li><a href="{% link gps/images/TURN_Network_poster_2019.pdf %}" rel="noopener" target="_blank">TURN GPS Network
+      <li><a href="{% link images/2023TURNGPSNetwork36x44.png %}" rel="noopener" target="_blank">TURN GPS Network
```

Comment:
after going through another pr and removing old images, let's try to keep these gps based images in the gps/images folder like they were. Do you mind moving them?

Reply:
done.